controllers/user.js

```diff
@@ -163,7 +163,8 @@ class Users {
     const tokenPayload = {
       id: user.id,
       isMentor: user.isMentor,
-      isAdmin: user.isAdmin
+      isAdmin: user.isAdmin,
+      username: user.username
```

Comment:
What feature on the frontend were you trying to achieve that prompted this inclusion?

Reply:
View Profile and update profile.

Reply:
Am using get a user to accomplish this task... and it takes username as params

Reply:
There is no other way I can get the username.

Reply:
Okay. We probably should have been getting the user by ID in the first place instead of the username, especially seeing as the ID will be more difficult to guess than the username.

I will approve this in the meantime, but propose that you guys should probably create a controller similar in fucntionality to the current one. It could be `getUserById` and will take the id from `decoded` instead of `params` to use in finding the user. If that is well done, you can then deprecate the one that gets user by username (or even leave it if you desire)

Reply:
ok... we will
thanks